Tools/yocto/rpi/bblayers.conf

```diff
@@ -5,6 +5,9 @@ POKY_BBLAYERS_CONF_VERSION = "2"
 BBPATH = "${TOPDIR}"
 BBFILES ?= ""
 
+# Needed for systemd-analyze
+DISTRO_FEATURES:append = " usrmerge"
+
```

Comment:
This is not the best place for it.
I suggest to put this instead in meta-webkit/conf/distro/webkitdevci.conf (via a patch) and also send a PR to meta-webkit

Reply:
Done (except for upstreaming). Will postpone upstreaming the meta-oe and meta-webkit patches, until I learned how to contribute their :-) Noted on my TODO list.

Reply:
Ok, that was easy, just made it:
https://github.com/Igalia/meta-webkit/pull/502
https://github.com/openembedded/meta-openembedded/pull/838